subsys/net/lib/downloader/Kconfig

```diff
@@ -26,6 +26,12 @@ config DOWNLOADER_MAX_FILENAME_SIZE
 	range 8 2048
 	default 255
 
+config DOWNLOADER_MAX_REDIRECTS
+	int "Default maximum number of redirects before download is aborted"
```

Comment:
It's not really the default once it's modified.. I'd just say: `Number of redirects ...`